sitemap.xml

```diff
@@ -28,10 +28,10 @@ category: hidden
       </url>
     {% endunless %}
   {% endfor %}
-  {% for post in site.posts %}
+  {% for post in site.posts %} {% comment %}Hard-coded URL here as variable wasn't working.{% endcomment %}
```

Comment:
Line 10 has
```
{{ site.a_rootpage }}{{ site.url }}
```

Reply:
I thought I tried that and it didn't work (returned null), but just tried it again and it does. OK will use that then :-)